### DISCUSSION TITLE
Utilities to Convert Django Forms to VDOM

### DISCUSSION THREAD

**Discussion by Archmonger** [reactive-python/reactpy-django#28] [Ideas]
It might be possible to develop utilities to convert [Django Forms](https://docs.djangoproject.com/en/3.2/topics/forms/) into a VDOM construct.

Django forms have methods to return HTML for all data attributes in the form, which we can easily create something to more conveniently plop on the page. We may also want to create some utilities that aggregates the data `onSubmit`, checks validity through the Django Forms API, and either performs a save or displays errors if they exist.

I'm not familiar with how [forms are accomplished in ReactJS](https://reactjs.org/docs/forms.html). But, converting Django forms to IDOM seems accomplishable without third party packages.

**Comment by rmorshea** [comment]
related: https://github.com/idom-team/idom/pull/859
